src/Compilers/CSharp/Test/Emit2/Diagnostics/DiagnosticSuppressorTests.cs

```diff
@@ -70,6 +70,72 @@ void M(int i)
             VerifySuppressedAndFilteredDiagnostics(compilation, analyzers);
         }
 
+        [WorkItem(62540, "https://github.com/dotnet/roslyn/issues/62540")]
+        [Fact]
+        public void TestSuppression_CompilerSyntaxBindingError_AndSuppressibleWarning()
+        {
+            const string SourceCode = @"
+                public class MyClass
+                {
+                    void MyPrivateMethod(int i)
+                    {
+                        // warning CS1522: Empty switch block
+                        // NOTE: Empty switch block warning is reported by the C# language parser
```

Comment:
This is no longer true. We should delete occurrences of this comment.

https://github.com/dotnet/roslyn/blob/77f1b7ac8756a08dbe5388a8f0dfa7f5eabd57bd/src/Compilers/CSharp/Portable/Binder/SwitchBinder_Patterns.cs#L35

Reply:
I have removed the `// NOTE: Empty switch block warning is reported by the C# language parser` comments that I added previously.